README.md

```diff
@@ -31,95 +33,71 @@ This proposal reached Stage 1 at the 2020 Feb TC39 meeting.
     - 1 hour, 46 minutes and 40 seconds → Wide
     - 1 hr, 46 min, 40 sec → Short
 
-# Quick Start Example
+## Quick Start
 
 ```javascript
 new Intl.DurationFormat("fr-FR", { style: "long" }).format({
     hours: 1,
     minutes: 46,
     seconds: 40,
 });
-// "1 heure, 46 minutes et 40 secondes"
+// => "1 heure, 46 minutes et 40 secondes"
 ```
 
-# V8 Prototypes
-Three v8 prototypes (try to use two different possible ICU classes) were made, ALL are
-* Sync with ["Stage 1 Draft / June 1, 2020" version of spec](https://github.com/tc39/proposal-intl-duration-format/commit/fc8ff131cf7e688810b38d7e95d6fa44b1f1964e)
-* Flag --harmony_intl_duration_format
-* Have not implement any changes not yet spec out in https://tc39.es/proposal-intl-duration-format/ such as
-  + hideZeroValued
-  + smallestUnit / largestUnit
+## Motivation
 
-1. Base the implementation on [icu::MeasureFormat::formatMeasures()](https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/classicu_1_1MeasureFormat.html#ade104d40578223bd194050914b090904)
-   * https://chromium-review.googlesource.com/c/v8/v8/+/2762664
-   * Not yet implment formatToParts
-   * Need solution of [ICU-21543 "Add methods to return FormattedValue to MeasureFormat "](https://unicode-org.atlassian.net/browse/ICU-21543) to implement formatToParts().
-2. Based on [the support of "-and-" unit in LocalizedNumberFormatter](https://unicode-org.github.io/icu-docs/apidoc/dev/icu4c/classicu_1_1number_1_1LocalizedNumberFormatter.html)
-   * https://chromium-review.googlesource.com/c/v8/v8/+/2775300
-   * Not yet implment formatToParts
-   * Not yet implement style:"digital"
-   * Need solution of the following to implement formatToParts():
-     + [ICU-21544 "unit format in number formatter return U_INTERNAL_PROGRAM_ERROR with "year-and-" (except month) and  "month-and-""](https://unicode-org.atlassian.net/browse/ICU-21544)
-     + [ICU-21547 "nextPosition() of FormattedNumber of unit with "-and-" is buggy"](https://unicode-org.atlassian.net/browse/ICU-21547)
-3. Based on icu::ListFormatter and icu::number::LocalizedNumberFormatter w/o depend on the "X-and-Y" units.
-   * https://chromium-review.googlesource.com/c/v8/v8/+/2776518
-   * Implements formatToParts
-    * Not yet implement style:"digital"
-# Motivation
-
-* Users need all types of duration format depending on the requirements of their application. For example, to show how long a flight takes, the duration should be in Short or Narrow format
-  + 1 hr 40 min 60 sec → Short
-  + 1 h 40 m 60 sec  → Narrow
+* Users need all types of duration formatting depending on the requirements of their application. For example, to show how long a flight takes, the duration should be in Short or Narrow format
+  + "1 hr 40 min 60 sec" → Short
+  + "1 h 40 m 60 s"  → Narrow
```

Comment:
```suggestion
  + "1h 40m 60s"  → Narrow
```

Narrow removes the spaces between numbers and units, right?

Reply:
See comment below.